README.md

```diff
@@ -83,7 +83,12 @@ To locally run the same UI as the Huggign Face space:
  
 1. Install the requirements in `src/requirements.txt` as well as `gradio`.
 2. Run `python src/app.py` and go to the local URL. From there you will be able to preview the papers from today, as well as the generated digests.
+3. If you want to run the action locally you can copy .env.template to .env and then set the environment variables in the .env file.
+- This file may be hidden by default in some operating systems due to the dot prefix. To reveal hidden files, follow the instructions for your 
+- The .env file is one of the files in .gitignore, this means that git does not track it and it will not be uploaded to the repository, accidentally.
+- For this reason you should not edit the original .env.template and put the keys in the original file, since the .template.env is tracked by git and editing it might cause you to commit it with your API keys.
```

Comment:
nit: not just the API Keys, but also their email address

Reply:
done